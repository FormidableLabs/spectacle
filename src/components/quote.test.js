import Quote from './quote';
import React from 'react';
import { mount } from 'enzyme';

describe('<Quote />', () => {
  test('should render correctly', () => {
    const context = {
      styles: {
        components: {
          quote: {
            color: '#ff0'
          }
        }
      }
    };
    const wrapper = mount(<Quote>Hello There!</Quote>, { context });
    expect(wrapper).toMatchSnapshot();
  });
});
