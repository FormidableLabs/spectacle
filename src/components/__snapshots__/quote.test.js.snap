// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Quote /> should render correctly 1`] = `
<Quote>
  <Styled(span)
    className={undefined}
    styles={
      Array [
        Object {
          "color": "#ff0",
        },
        Object {},
        Object {},
        undefined,
      ]
    }
  >
    <span
      className="css-tr63ps"
    >
      Hello There!
    </span>
  </Styled(span)>
</Quote>
`;
